src/heading.js

```diff
@@ -86,10 +86,18 @@ export default class Heading extends Plugin {
 			...getCommandsBindingTargets( commands, 'value' ),
 			// ...and chose the title of the first one which #value is true.
 			( ...areActive ) => {
-				const index = areActive.findIndex( value => value );
-
-				// If none of the commands is active, display default title.
-				return options[ index ] ? options[ index ].title : defaultTitle;
+			  const index = areActive.findIndex( value => value );
+
+			  // If none of the commands is active, display default title.
+              let title = defaultTitle;
+              if (options[index]) {
+                if (options[index].viewElement) {
+                  title = options[index].viewElement.toUpperCase();
+                } else {
+                  title = "P";
```

Comment:
nit: do we need to refactor this?

Reply:
What do you mean?

Reply:
I think Ryan wanted to have this say `Aa` or `aA`, @ryanfeeley do you remember?

Reply:
This code uses Aa when nothing is selected P, H3, H2, H1 when something is. However my previous commit was using Aa all the time.